gotpm/cmd/utils.go

```diff
@@ -85,3 +94,17 @@ func dataInput() io.Reader {
 	}
 	return file
 }
+
+// Get the algorithm for public key.
+func getAlgo(keyAlgo string) (tpm2.Algorithm, error) {
+	switch keyAlgo {
+	case "rsa":
+		return tpm2.AlgRSA, nil
+	case "ecc":
+		return tpm2.AlgECC, nil
+	case "":
```

Comment:
I made a mistake adding this `case`. Just handle `""` in the default.

Reply:
removed case ""